src/Symfony/Component/Yaml/Parser.php

```diff
@@ -359,6 +299,31 @@ public function parse($value, $flags = 0)
             mb_internal_encoding($mbEncoding);
         }
 
+        if (array_key_exists('<<', $data)) {
```

Comment:
Indeed, that's a valid point. However, I think we should do that in a different PR (and this should then also be documented in the upgrade files).